src/app/theme/ThemeProvider.tsx

```diff
@@ -79,6 +89,13 @@ export const ThemeProvider: React.FC<ThemeProviderProps> = ({ children }) => {
         root.style.backgroundRepeat = 'no-repeat';
         document.documentElement.classList.add('dark');
         setCheckoutTheme('dark');
+      } else if (currentTheme === 'idmanagement') {
+        root.style.backgroundImage = `url(${IdManagementBG})`;
+        root.style.backgroundSize = 'cover';
+        root.style.backgroundPosition = 'center';
+        root.style.backgroundRepeat = 'no-repeat';
+        document.documentElement.classList.add('dark');
+        setCheckoutTheme('dark');
```

Comment:
I say the same as in the comment above, if this has to be deployed today, it can be left as it is, but this is becoming unsustainable. This can be substantially improved by for example adding the style parameters in a constant with the theme configuration and applying that configuration in a few lines of code. 
This will make it much easier to maintain